src/operation/manipulatefilters.py

```diff
@@ -135,6 +135,88 @@ def process(self, db, user, filter_id):
 
         return OperationResult()
 
+class ReapplyReviewFilters(Operation):
+    def __init__(self):
+        Operation.__init__(self, {"review_id": int, "dry_run": Optional(bool)})
+
+    def process(self, db, user, review_id, dry_run=False):
+        if user.isAnonymous():
+            return OperationFailureMustLogin()
+
+        review = dbutils.review.Review.fromId(db, review_id)
+
+        if not user.hasRole(db, "administrator") and not user in review.owners:
+            raise OperationFailure(
+                code="notallowed",
+                title="Not allowed!",
+                message="Operation not permitted.")
+
+        applyfilters = review.applyfilters
+        applyparentfilters = review.applyparentfilters
```

Comment:
I think you don't have to use these two variables above. You can use values directly below